.changeset/tired-wasps-fry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-documentation': patch
+---
+
+- Updated the main Figma link in the `Introduction` page
+- Restored the list of HTML components in the `Getting Started > Developer` page
```

Comment:
```suggestion
- Restored the list of HTML components in the `Getting Started > Developer` page

```
Otherwise it will be incorrectly formatted in the changeset PR